code/game/objects/items/weapons/power_cells.dm

```diff
@@ -18,6 +18,17 @@
 	var/minor_fault = 0 //If not 100% reliable, it will build up faults.
 	var/chargerate = 1000 //how much power is given every tick in a recharger
 	var/self_recharge = 0 //does it self recharge, over time, or not?
+	var/grown_battery = FALSE // If it's a grown that acts as a battery, add a wire overlay to it.
+
+/obj/item/weapon/stock_parts/cell/New()
+	..()
+	processing_objects.Add(src)
+	charge = maxcharge
+	updateicon()
+
+/obj/item/weapon/stock_parts/cell/Destroy()
+	processing_objects.Remove(src)
+	return ..()

```

Comment:
Basically, some cells, that auto-generate charge are in the `processing_objects` list and need to be removed from said list if they are auto-charging.

Any cell can be auto-charging; not just slime cells.